teabot/worker.py

```diff
@@ -25,8 +33,8 @@ def do_work():
     server_link.send_queued_update_if_time()
     global last_status, last_number_of_cups
 
-    current_weight = weight_sensor.get_reading()
-    temperature = temperature_sensor.get_reading()
+    current_weight = weight_sensor.get_readings(wait=True)[-1]['reading']
```

Comment:
Not sure how I feel about `get_readings(wait=True)[-1]['reading']` should this be hidden behind something like `get_latest_reading()` ?